### ISSUE TITLE
Testing Methods in Application 'start'

### ISSUE THREAD

**Issue by nartz** [TestFX/TestFX#408]
Hi - 
I'm trying to figure out the right way to set up the UI to be tested. I want to load the entire UI, not just a single pane. The UI is loaded using FXML. The issue I'm having is that I find currently i have to copy paste the Main#start method from my application into MainTest#start fully, for instance, things like this setOnCloseRequest listener, need to be set on the main stage in Main#start:

stage.setOnCloseRequest(new EventHandler<WindowEvent>() {
                @Override
                public void handle(WindowEvent e) {
                   //..implementation...
                }
            });
So because this is on the main stage, I'm unsure of how to set it in a TestFx ApplicationTest#start without simply copy pasting it.

In a nutshell - is there an easier way rather than copy pasting my existing Main#start into MainTest#start?

**Comment by JordanMartinez** [comment]
I believe you can easily launch your own application if you set up a test class much like the one they did [here](https://github.com/TestFX/TestFX/blob/master/subprojects/testfx-junit5/src/test/java/org/testfx/cases/acceptance/ApplicationLaunchTest.java)

**Comment by JordanMartinez** [comment]
@brcolow This is another one that I think can be closed.
